src/schedule/ll_schedule.c

```diff
@@ -64,13 +64,19 @@ DECLARE_TR_CTX(ll_tr, SOF_UUID(ll_sched_uuid), LOG_LEVEL_INFO);
  *                         +---------+
  */
 
+struct ll_dsp_load {
+	unsigned int cycles_sum;
+	unsigned int checks;
+};
+
```

Comment:
I would like to suggest to track peak value as well and add cycles_peak field. The average value may hide random peaks that can cause LL delays and trigger xruns.